### PR TITLE
Use CNI 0.8.5 for Kubernetes 1.18+

### PR DESCRIPTION
**Pull request by hakman**
Equivalent of https://github.com/kubernetes/kubernetes/pull/78819.

### CODE REVIEW

File: upup/pkg/fi/cloudup/networking.go [hunk 1]
```diff
@@ -160,19 +164,23 @@ func findCNIAssets(c *api.Cluster, assetBuilder *assets.AssetBuilder) (*url.URL,
...
+	if util.IsKubernetesGTE("1.18", *sv) {
+		cniAsset = defaultCNIAssetK8s1_18
+		cniAssetHash = defaultCNIAssetSHA256StringK8s1_18
+		klog.V(2).Infof("Adding default CNI asset for k8s >= 1.18: %s", cniAsset)
+	} else if util.IsKubernetesGTE("1.15", *sv) {
 		// We're still on the same asset, but we use sha256
 		cniAsset = defaultCNIAssetK8s1_11
 		cniAssetHash = defaultCNIAssetSHA256StringK8s1_11
-		klog.V(2).Infof("Adding default CNI asset for k8s >= 1.11: %s", defaultCNIAssetK8s1_9)
+		klog.V(2).Infof("Adding default CNI asset for 1.18 > k8s >= 1.11: %s", cniAsset)
```
Comment (rifelpet) [inline upup/pkg/fi/cloudup/networking.go:175]:
```suggestion
		klog.V(2).Infof("Adding default CNI asset for 1.18 > k8s >= 1.15: %s", cniAsset)
```

File: upup/pkg/fi/cloudup/networking.go [hunk 2]
```diff
@@ -160,19 +164,23 @@ func findCNIAssets(c *api.Cluster, assetBuilder *assets.AssetBuilder) (*url.URL,
...
 	} else if util.IsKubernetesGTE("1.11", *sv) {
 		cniAsset = defaultCNIAssetK8s1_11
 		cniAssetHash = defaultCNIAssetSHA1StringK8s1_11
-		klog.V(2).Infof("Adding default CNI asset for k8s >= 1.11: %s", defaultCNIAssetK8s1_9)
+		klog.V(2).Infof("Adding default CNI asset for 1.18 > k8s >= 1.11: %s", cniAsset)
```
Comment (rifelpet) [inline upup/pkg/fi/cloudup/networking.go:179]:
```suggestion
		klog.V(2).Infof("Adding default CNI asset for 1.15 > k8s >= 1.11: %s", cniAsset)
```

File: upup/pkg/fi/cloudup/networking.go (hunk 1 above)
Comment (hakman) [reply upup/pkg/fi/cloudup/networking.go:175]: It is actually the same version, just the hash changed from sha1 to sha256, right?

Comment (rifelpet) [reply upup/pkg/fi/cloudup/networking.go:175]: ah you're right, nevermind

**Pull request merged**